README.md

```diff
@@ -1 +1,72 @@
-# strong-error-handler
\ No newline at end of file
+# strong-error-handler
+
+Error handler for use in development (debug) and production environments.
+
+ - When run in production mode, error responses are purposely undetailed
```

Comment:
@bajtos 
"details of the error response are purposely removed"

"undetailed" is not a word.

I can make the change in a new PR if you agree to it.